internal/xds/translator/testdata/in/xds-ir/listener-connection-limit.yaml

```diff
@@ -44,26 +44,28 @@ tcp:
   connection:
     limit:
       value: 3
-  tls:
-    passthrough:
-      snis:
-      - bar.com
-  destination:
-    name: "tls-route-dest"
-    settings:
-    - endpoints:
-      - host: "1.2.3.4"
-        port: 50000
+  routes:
```

Comment:
This is a bug in the listener-connection-limit test. The TCPRoute structure is incorrect.